gulpfile.js

```diff
@@ -0,0 +1,239 @@
+const argv = require('yargs');
+const bump = require('gulp-bump');
+const exec = require('child_process').exec;
+const del = require('del');
+const es = require('event-stream');
+const fs = require('file-system');
+const git = require('gulp-git');
+const gulp = require('gulp');
+const debug = require('gulp-debug');
+const inlinesource = require('gulp-inline-source');
+const moment = require('moment');
+const shell = require('gulp-shell');
+
+//Parameters and Variables
+const params = require('./build/parameters');
+let lerna = require('./lerna.json');
+
+function adoPrep(done) {
+    var branchName = params.build.buildSourceBranch;
+    if (branchName !== 'master') {        
+        branchName = branchName.replace(/refs\/heads\/(feature\/)?/i, '');
+    }    
+    var gitScript = `sudo git checkout ${branchName}`;
+    console.log('ADO Prep Script: ' + gitScript);
+    return runCmd(gitScript, done);    
+}
+
+function bumpVersion() {
+    return gulp.src(['ingredient/**/package.json','core/package.json','system/package.json','lerna.json'], { base: './' })
```

Comment:
Do we want gulp to bump versions? Doesn't lerna do that for us?

Reply:
bump version was for circumstances outside of lerna/npm bumping versions. AKA if a build borks and we need to bump the version outside of modifying and committing new versions.

Reply:
removed.